dbt/adapters/bigquery/connections.py

```diff
@@ -240,21 +242,21 @@ def format_rows_number(self, rows_number):
         return f"{rows_number:3.1f}{unit}".strip()
 
     @classmethod
-    def get_bigquery_credentials(cls, profile_credentials):
+    def get_bigquery_credentials(self, profile_credentials):
```

Comment:
Not necessary, you [have to pass](https://docs.python.org/3/library/functions.html#classmethod
) `cls` instead of `self` in a `@classmethod`.